test/factories/project_factory.py

```diff
@@ -31,6 +31,12 @@ def _create(cls, model_class, *args, **kwargs):
             project.set_password('hello')
         if 'data_classification' not in project.info:
             project.info['data_classification'] = dict(input_data="L4 - public", output_data="L4 - public")
+        if 'kpi' not in project.info:
+            project.info['kpi'] = 0.5
+        if 'product' not in project.info:
+            project.info['product'] = 'exists'
```

Comment:
We may need to update this to a matching value from the config (product `abc`, and subproduct `def`).